src/core/cat.js

```
@@ -68,7 +79,7 @@ function main({ args = [], stdin = '', env = {} }) {
     if (showEnds) s += '$';
     let prefix = '';
     if ((numberNonblank && content !== '') || numberAll) {
-      prefix = String(lineNum).padStart(6, ' ') + '\t';
+      prefix = String(lineNum).padStart(numWidth, ' ') + '\t';
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)